java-checks/src/test/files/checks/ClassFieldCountCheck/UnusualTrees.java

```diff
@@ -0,0 +1,26 @@
+enum TooManyFieldsEnum { // Noncompliant [[sc=6;ec=23]] {{Refactor this class so it has no more than 2 fields, rather than the 3 it currently has.}}
+  ENUM_CONSTANT_TREE { // Noncompliant [[sc=3;ec=4]] {{Refactor this class so it has no more than 2 fields, rather than the 3 it currently has.}}
```

Comment:
Same remark than for the anonymous class. we should only highlight the name. When different lines are involved, please use the `endLine` specifier (`[[sc=3;el=+4;ec=4]]`).